tests/bare/result

```diff
@@ -0,0 +1 @@
+SUCCESS
```

Comment:
Whats the purpose of this file?

Reply:
It is generated in case of success of the training, it should not be committed, I will remove it